pyScienceMode2/sciencemode.py

```diff
@@ -126,13 +128,16 @@ def __init__(self, port: str, show_log: bool | str = False, with_motomed: bool =
         self.__watchdog_thread_started = False
         self.command_send = []  # Command sent to the rehastim
         self.ack_received = []  # Command received by the rehastim
-        self.TypeReha2 = TypeReha2
-        self.TypeRehap24 = TypeRehap24
+
+        from pyScienceMode2 import Rehastim2Commands, RehastimP24Commands
```

Comment:
Move the import in the import section

Reply:
If I decide to import the classes directly into the init file, I'll get a lot of circular import errors, and I've seen that one solution is to import locally.